compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -422,6 +426,15 @@ internal class ComposeContainer(
     fun createComposeSceneContext(platformContext: PlatformContext): ComposeSceneContext =
         ComposeSceneContextImpl(platformContext)
 
+    private fun updateLifecycleState() {
```

Comment:
Not a big deal, but I'd like to see these flags encapsulated (maybe `PlatformWindowContext` is a good place since it's all about window state) and use approach like in `copy`:
```kt
fun setLifecycleState(
    isDisposed: Boolean = this.isDisposed,
    isDetached: Boolean = this.isDetached,
    isMinimized: Boolean = this.isMinimized,
    isFocused: Boolean = this.isFocused,
) { ... }
```